packages/pilot/src/components/MetricList/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import shortid from 'shortid'
+import {
+  Card,
+  CardTitle,
+  CardContent,
+} from 'former-kit'
+
+import MetricListItem, { listItemPropTypes } from './MetricListItem'
+
+import styles from './style.css'
+
+const MetricList = ({
+  items,
+  title,
+}) => (
+  <Card>
+    <CardTitle title={title} />
+    <CardContent
+      className={styles.list}
+    >
+      {items.map(({
+        icon,
+        title: itemTitle,
+        value,
+      }) => (
+        <MetricListItem
+          key={shortid.generate()}
+          icon={icon}
+          title={itemTitle}
+          value={value}
+        />
+      ))}
+    </CardContent>
+  </Card>
+)
+
+MetricList.propTypes = {
+  items: PropTypes.arrayOf(PropTypes.shape(listItemPropTypes)).isRequired,
```

Comment:
aqui pode usar dessa forma pra não precisar exportatr os proptypes no arquivo do `MetricListItem`
```suggestion
  items: PropTypes.arrayOf(PropTypes.shape(MetricListItem.propTypes)).isRequired,
```

Reply:
Corrigido!